metadata.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "puppet-system",
-  "version": "0.8.3-rc0",
+  "version": "0.8.4",
```

Comment:
Please don't update the version. We will do this during our deployment of the next version.